.github/workflows/check.yml

```diff
@@ -8,12 +8,12 @@ on:
 
 jobs:
   check:
-    uses: wetransform/gha-workflows/.github/workflows/gradle-library-check.yml@ef7e7ff0d5f77914d591c783f3dcf854e3c4e204
+    uses: wetransform/gha-workflows/.github/workflows/gradle-library-check.yml@2a6fc2ee6aeac10bafcb841f1be6f780f2b5dac6
```

Comment:
It seems when just using the commit sha here renovate will ignore it. I would prefer if we can let renovate watch for updates. I guess for that then the notation similar to [here](https://github.com/halestudio/hale-cli/blob/b1559b1cd0e7d731c8bdbad2af9204558439d224/.github/workflows/stale.yml#L11) is needed.

Would you do the honor and [create a release for the workflows](https://github.com/wetransform/gha-workflows/actions/workflows/tf-release.yml) so that you can use that version?

Reply:
> Would you do the honor and [create a release for the workflows](https://github.com/wetransform/gha-workflows/actions/workflows/tf-release.yml) so that you can use that version?

Created the release.

> It seems when just using the commit sha here renovate will ignore it. I guess for that then the notation similar to [here](https://github.com/halestudio/hale-cli/blob/b1559b1cd0e7d731c8bdbad2af9204558439d224/.github/workflows/stale.yml#L11) is needed.

Looks like it picked it up, there's [this PR](https://github.com/halestudio/hale-cli/pull/126/files) now. It uses the hash from the release but doesn't mention the version number, though.

Reply:
I think it would be better to add the version numbers, then it can properly differentiate between a major update and other updates. And presumably only reacts to releases and not every push to `master`.

Reply:
I created PR #127 for it